spec/lib/hackney/income/view_my_cases_spec.rb

```diff
@@ -148,6 +148,40 @@
           expect(subject.cases.count).to eq(1)
         end
       end
+
+      context 'when filtering cases by patch' do
+        subject {
+          view_my_cases.execute(
+            user_id: user_id,
+            page_number: page_number,
+            number_per_page: number_per_page,
+            patch: patch
+          )
+        }
+
+        let(:patch) { Faker::Lorem.characters(3) }
+
+        it 'returns only cases in specified patch' do
```

Comment:
This spec description misled me - as I understand it, this is actually testing the collaboration between this unit and another unit.

I'd use this language in acceptance specs, as that's what I'm confirming works. For this - which strikes me more as a unit test - I'd express what messages I'm expecting it to send to its (fake) collaborators:

```suggestion
        it 'asks the gateway for cases filtered by patch' do
```